doc/cli.markdown

```diff
@@ -1475,6 +1485,11 @@ This feature requires a device running supervisor version v9.7.0 or greater.
 
 Don't tail application logs when pushing to a local mode device
 
+#### --service &#60;service&#62;
+
+Only show logs from a single service.
+Only valid when pushing to a local mode device.
```

Comment:
Doesn't this also work for non local devices?
If it doesn't the example above is wrong:
```
balena logs 23c73a1 --service my-service
```

Reply:
Oh yeah, forgot to take this out... Good catch!

Reply:
Wait, no, both the documentation and example are correct, logs are only displayed when pushing to a local mode device. The section you are referencing is for push, not logs.